_data/settings.yml

```diff
@@ -0,0 +1,321 @@
+#-------------------------------
+# General Site Settings
+
+title: Project Flotta
+logo: assets/images/flotta-logo.png
+author: themefisher
+description: this is meta description
+email: flotta@redhat.com
+phone:
+address:
+copyright: Copyright &copy; 2020 Designed by [Themefisher](https://themefisher.com) & Developed by [Getjekyllthemes](https://getjekyllthemes.com)
+
+menu:
+# - {name: About, link: '/about'}
+- {name: Blog, link: '/blog.html'}
+- {name: Documentation, link: '/documentation'}
+# - {name: Portfolio, link: '/portfolio'}
+# - {name: Contact, link: '/contact'}
+
+
+#-------------------------------
+# Google Analytics Section
+
+# Analytics helps you understand who visits your website.
+# Learn more at: http://support.google.com/analytics/answer/1008065
+
+# NOTE: If you want to use Google Analytics for basic web tracking: Just uncomment and enter your UA-Code here.
+
+google-analytics: # UA-XXXXXX-X
+
+
+#-------------------------------
+# Comment Section (Disqus)
+
+disqus-identifier: # Add your shortname for Disqus Comment. For example mr-brown
+
+
+#-------------------------------
+# Hero Section
+
+hero:
+  hero-title: Project Flotta. Edge device management for k8s workloads.
+  hero-text: Manage, deploy, observe edge workloads from a single k8s instance. 
+  hero-image: assets/images/flotta-device-diagram.png
+social:
+# - {icon: "ti-facebook", link: "#"}
+# - {icon: "ti-instagram", link: "#"}
+# - {icon: "ti-dribbble", link: "#"}
+# - {icon: "ti-twitter", link: "#"}
+- {icon: "ti-github", link: "https://github.com/project-flotta/"}
+
+
+#-------------------------------
+# About (Section and Page)
+
+# About Section
+blurb:
+  Project Flotta is designed to manage containerized workload on small-footprint devices found on the edges of networks by managing the lifecycle of edge-focused operating systems and securing connectivity between K8s/OCP and the edge. Flotta provides a pipeline for operating system images across ARM and x86 architectures and enables atomic upgrades and rollbacks of an edge-focused operating system, while maintaining a minimal footprint. Because edge computing networks have the potential for instability, Flotta can provide continuity when connectivity is sporadic or down for long periods of time. Enhanced edge observability is also provided through improved metrics and log collection capabilities.
+
+blurb-btn-link: '/documentation'
+blurb-btn-name: know more
+
+
+# About Page
```

Comment:
Should we update about page?

Reply:
I kept as reference in the future if needed.